content/authors/adkris/_index.md

```diff
@@ -66,11 +65,11 @@ user_groups:
 - Researchers
 ---
 
-I am a student at the University of Washington majoring in Electrical Engineering with a concentration in Digital Signal Processing. I am expected to graduate the Spring of 2024 and am interested in pursuing graduate research after my bachelor's degree.
+I am a student at the University of Washington majoring in Electrical Engineering and pursuing Digital Signal Processing and Neural Engineering concentrations. I am expected to graduate the Spring of 2024 and am interested in pursuing graduate research after my bachelor's degree.
 
-My core competency is in programming, with an expertise in Java and Python. I aim to use my programming skills to aid in ecological & environmental conservation. 
+My core competency is in programming, with an expertise in Python and Java. I am currently studying how signal processing methods are used in passive acoustic monitoring projects to address project-specific goals and aid in ecological conservation.
```

Comment:
```suggestion
My core competency is in programming, with an expertise in Python and Java. I am currently applying signal processing techniques in passive acoustic monitoring of bats with a goal toward (?) ecological conservation.
```
I shortened the sentence a bit, but not sure about the goal part, hence the "?"

Reply:
Honestly, I'll keep it as "toward ecological conservation" but the more I think about it, I am starting to see bats as a very interesting system to study on multiple levels (echolocation, bioindicators, social dynamics) so my goal feels really to just study bats haha.